### ISSUE TITLE
[NTR] add "Raman scattering" and its subtypes according to ISO specs

### ISSUE THREAD

**Issue by StroemPhi** [NFDI4Chem/VibrationalSpectroscopyOntology#103]
|Preferred term label|Synonyms|Textual definition|Suggested parent term|Link to [CHARISMA Wiki equivalent](https://wiki.charisma.ideaconsult.net/wiki/List_of_all_terms)|Examples|Attribution|
|---|---|--------------|---|---|---|---|
|Raman scattering|Raman effect|ISO 18115-3: inelastic scattering of electromagnetic radiation, characterized by a loss or gain in energy of the incident photons corresponding to a energy transition within the scatterer|it needs to be decided whether this should be modeled according to a more general pattern as "natural process" which gets utilized in a special kind of Raman Spectroscopy assay. Or whether these to can be conflated into only the latter. Philip's first guess would be to go with the first approach and the CHARISMA wiki definitions support this so far. (briefly discussed on the 11-08-2023 call)|https://wiki.charisma.ideaconsult.net/wiki/Item:Q28|---|---|
|anti-Stokes Raman scattering|---|ISO 18115-3: Raman scattering (3.5.16) in which the emitted photon has higher energy than the incident photon|Raman scattering|doesn't exist yet|---|---|
|coherent anti-Stokes Raman scattering|---|--------------|---|---|---|---|
....

# TODO: add also all the other types of scatterings mentioned in the ISO specs to this NTR

**Comment by Zack-83** [comment]
Our authoritative sources: [CHMO](https://www.ebi.ac.uk/ols/ontologies/chmo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCHMO_0000656#), [ISO18115:3](https://drive.google.com/file/d/1xM0vNvgf25k9QXB1hxB6l0E3XfWvQL4Z/) and [CHARISMA](https://wiki.charisma.ideaconsult.net/wiki/List_of_all_terms), contain an overview of Raman sub-techniques (assays).

Neither of these is complete nor strictly coherent. Therefore it is our task to establish a classification of techniques and include it into VIBSO (or propose it to CHMO).

To put order within the jungle, we need to define some positive criteria, e.g.:
* techniques defined by one **distinctive** adjective (e.g. coherent vs non-coherent)
* techniques defined by an **established** combination of two or more adjectives (e.g. SERS, CARS)
* techniques which imply a specific set of experimental parameter

as well as some negative criteria:
* techniques defined by a **casual** combination of two or more adjectives (e.g. deep ultra-violet resonance Raman spectroscopy)
* techniques described in relation to aspects independent from the acquisition method such as sample preparation (e.g. drop coating deposition Raman spectroscopy)

unsure:
* techniques whose denomination does not differentiate it from the default (e.g. laser Raman spectroscopy: nowadays no experiment is conducted without a laser)
* techniques which do not imply different parameters (e.g. infrared vs ultra-violet vs visible)
* techniques which are too seldom, old or restricted (e.g. transmission Raman spectroscoy)

The combination techniques should be included axiomatically, by allowing instances to belong to more classes:

```ttl
# An example of surface-enhanced anti-Stokes confocal Raman spectroscopy
ex:Measurement1 rdf:type vibso:RamanSpec_SurfaceEnhanced , # implies parameters about the surface
                         vibso:RamanSpec_AntiStokes ,      # implies a restriction on Raman shifts <0
                         vibso:RamanSpec_Confocal .        # implies parameters on the focal depth...
```

<details><summary><h2>Terms to add</h2></summary>

|ID          |label          |alternative term|definition|notes|parent element        |
|------------|---------------|----------------|----------|-----|----------------------|
|ISO18115:3.5.16|Raman scattering|Raman effect|[def16]|[16]|inelastic light scattering|
|CHMO:0000656; ISO18115:3.5.17; CHARISMA:Q87|Raman spectroscopy||[def17]|[17]|optical spectroscopy ∩ scattering spectroscopy (< assay)|
|                            |Raman microscopy            ||       ||optical microscopy (< assay) |
|CHMO:0000663                |confocal Raman spectroscopy ||       ||Raman spectroscopy|
|CHMO:0000664; ISO18115:3.5.06|confocal Raman microscopy  ||[def06]||Raman microscopy  |
|CHMO:0001986           |non-resonance Raman spectroscopy ||       ||Raman spectroscopy|
|CHMO:0000668; ISO18115:3.5.20; CHARISMA:Q234|resonance Raman spectroscopy|RR|[def20]||Raman spectroscopy|
|CHMO:0000677; ISO18115:3.5.26|surface-enhanced Raman spectroscopy|SERS|[def26]|[26]|Raman spectroscopy|
|CHMO:0000678; ISO18115:3.5.30|tip-enhanced Raman spectroscopy    |TERS|[def30]|[30]|Raman spectroscopy|
|CHMO:0000670; ISO18115:3.5.29|surface-enhanced resonant Raman spectroscopy|SERRS|[def29]|[29]|resonance ∩ surface-enhanced Raman spectroscopy|
|ISO18115:3.5.31              |tip-enhanced resonance Raman spectroscopy   |TERRS|[def31]|    |resonance ∩ tip-enhanced Raman spectroscopy    |
|            |Stokes Raman spectroscopy                   |    |||Raman spectroscopy|
|            |anti-Stokes Raman spectroscopy              |    |||Raman spectroscopy|
|CHMO:0000658|coherent Raman spectroscopy                 |CRS |||Raman spectroscopy|
|CHMO:0000659|coherent Stokes Raman spectroscopy          |CSRS|||coherent ∩ Stokes Raman spectroscopy|
|CHMO:0000661|coherent anti-Stokes Raman spectroscopy|CARS|||coherent ∩ anti-Stokes Raman spectroscopy|
|ISO18115:3.5.22; CHARISMA:Q236|spontaneous Raman spectroscopy||[def22]|[22]|Raman spectroscopy|
|CHMO:0000675; CHARISMA:Q237|stimulated Raman spectroscopy |SRS|||Raman spectroscopy|
|CHMO:0001939         |stimulated Raman gain spectroscopy |    |||stimulated Raman spectroscopy|
|CHMO:0001940         |stimulated Raman loss spectroscopy |    |||stimulated Raman spectroscopy|
|CHMO:0000657                 |chiral Raman spectroscopy  |    |||Raman spectroscopy|
|ISO18115:3.5.23; CHARISMA:Q235|spatially offset Raman spectroscopy|SORS|[def23]|[23]|Raman spectroscopy|
|CHMO:0000667; ISO18115:3.5.11; CHARISMA:Q117|polarized Raman spectroscopy|polarised Raman spectroscopy|[def11]|[11]|Raman spectroscopy|
</details>

<details><summary><h2>Unsure</h2></summary>

|ID             |label               |alternative term|definition|notes|parent element         |
|---------------|--------------------|----------------|----------|-----|-----------------------|
|CHMO:0001766; ISO18115:3.5.32|transmission Raman spectroscopy||[def32]|[32]|Raman spectroscopy|
|ISO18115:3.5.14|Raman mapping|Raman imaging|[def14]|[14]|Raman microscopy ∩ mapping technique|
|ISO18115:3.5.13|Raman depth profiling |    |[def13]|[13]|Raman mapping                       |
|ISO18115:3.5.04|coherent Raman scattering microscopy|CRS|[def04]|[04]|Raman microscopy ∩ coherent microscopy|
|ISO18115:3.5.35|vibrational-rotational spectroscopy |   |[def35]|[35]  |optical spectroscopy|
|ISO18115:3.5.21|rotational Raman spectroscopy       |   |[def21]|      |Raman spectroscopy  |
|               |electronic Raman spectroscopy       |   |       |      |Raman spectroscopy  |
|CHMO:0000870   |near-infrared Raman spectroscopy    |   |       |      |Raman spectroscopy  |
|CHMO:0000680   |ultraviolet Raman spectroscopy      |   |       |      |Raman spectroscopy  |
|CHMO:0000656   |laser Raman spectroscopy            |   |       |      |Raman spectroscopy  |
|CHMO:0000666   |hyper Raman spectroscopy  ||||Raman spectroscopy OR scattering spectroscopy |
||back focal plane imaging|Fourier space imaging, Fourier-plane imaging|||Raman spectroscopy |
</details>

<details><summary><h2>Terms to be dropped</h2></summary>

Techniques not fulfilling the criteria:
|ID           |label                                                |
|-------------|-----------------------------------------------------|
|CHMO:0000671 |ultraviolet resonance Raman spectroscopy             |
|CHMO:0000672 |deep ultra-violet resonance Raman spectroscopy       |
|CHMO:0000662 |coherent anti-Stokes resonance Raman spectroscopy    |
|CHMO:0000660 |coherent Stokes resonance Raman spectroscopy         |
|CHMO:0000669 |Kerr-gated resonance Raman spectroscopy              |
|CHMO:0000885 |chiral surface-enhanced Raman spectroscopy           |
|CHMO:0002373 |surface-enhanced non-resonance Raman spectroscopy    |
|CHARISMA:Q239|Surface-enhanced spatially offset Raman spectroscopy |
|CHARISMA:Q238|Surface plasmon polariton enhanced Raman spectroscopy|
|CHMO:0000875 |electrochemical surface-enhanced Raman spectroscopy  |
|CHMO:0000770 |single molecule surface-enhanced Raman spectroscopy  |
|CHMO:0000679 |ultra-high vacuum tip-enhanced Raman spectroscopy    |
|CHMO:0002165 |drop coating deposition Raman spectroscopy           |
|CHMO:0001943 |spectral excitation higher-order Raman spectroscopy  |
|CHMO:0001767 |optical tweezers Raman spectroscopy                  |
|CHMO:0000675 |stimulated Raman spectroscopy                        |
|CHMO:0001942 |ionisation-detected stimulated femtosecond Raman spectroscopy|
|CHMO:0001941 |photoacoustic Raman spectroscopy                     |
|CHMO:0000665 |Fourier transform Raman spectroscopy                 |

Phenomena listed in ISO18115, converted to techniques:
|ID             |label       |alternative term|definition|notes        |
|---------------|------------|----------------|----------|-------------|
|ISO18115:3.5.08|electronic Raman scattering |   |[def08]|             |
|ISO18115:3.5.24|stimulated Raman scattering |SRS|[def24]|[24-1] [24-2]|
|ISO18115:3.5.25|Stokes Raman scattering     |   |[def25]|[25]         |
|ISO18115:3.5.01|anti-Stokes Raman scattering|   |[def01]|             |

Phenomena listed in ISO18115, for which the equivalent technique is already defined:
|ID             |label                 |alternative term|definition|notes|
|---------------|----------------------|----------------|----------|-----|
|ISO18115:3.5.05|coherent Stokes Raman scattering     |CSRS|[def05]|[05-1] [05-2] [05-3] [05-4]|
|ISO18115:3.5.03|coherent anti-Stokes Raman scattering|CARS|[def03]|[03-1] [03-2] [03-3] [03-4]|
|ISO18115:3.5.27|surface-enhanced Raman scattering    |SERS|[def27]|[27-1] [27-2] [27-3]|
|ISO18115:3.5.28|surface-enhanced resonant Raman scattering|SERRS|[def28]|[28]|
|ISO18115:3.5.10|hyper-Raman scattering                    |     |[def10]|    |
</details>

<details><summary><h2>Other terms which do not describe a technique (to be moved to another issue)</h2></summary>

Configurations:
|ID             |label   |alternative term|definition|notes|parent element       |
|---------------|--------|----------------|----------|-----|---------------------|
|ISO18115:3.5.02|backscattering configuration||[def02]|[02]|optical configuration|
|ISO18115:3.5.34|transmission configuration  ||[def34]|[34]|optical configuration|

Parameters/quantities:
|ID             |label  |alternative term|definition|notes|parent element   |
|---------------|-------|----------------|----------|-----|-----------------|
|ISO18115:3.5.19|Raman shift                ||[def19]|[19]|energy difference|
|ISO18115:3.5.07|depolarization ratio       ||[def07]|    |intensity ratio  |
|ISO18115:3.5.09|enhancement factor||[def09]|[09-1] [09-2]|intensity ratio  |
|ISO18115:3.5.12|Raman amplification        ||[def12]|[12]|intensity ratio  |
|ISO18115:3.5.15|Raman optical activity  |ROA|[def15]|[15]|optical activity |
|ISO18115:3.5.18|Raman tensor               ||[def18]|    |tensor           |
|ISO18115:3.5.33|resonant excitation profile||[def33]|    |                 |
</details>

[CHARISMA Wiki equivalent]: https://wiki.charisma.ideaconsult.net/wiki/List_of_all_terms
[def01]: ## "Raman scattering (3.5.16) in which the emitted photon has higher energy than the incident photon"
[def02]: ## "optical configuration where the collected scattered light is in the opposite direction of the incident light"
[def03]: ## "multi-photon, third order nonlinear, wave mixing process that enhances the anti-Stokes Raman signal"
[def04]: ## "family of multi-photon microscopy techniques which use Raman active scattering modes of molecules"
[def05]: ## "multi-photon, third order nonlinear, wave mixing process that enhances the Stokes Raman signal"
[def06]: ## "Raman spectroscopy performed via a confocal optical microscopy set-up, in which a well-characterized mono-chromatic light is used to excite a sample and Raman scattered light is collected"
[def07]: ## "intensity ratio of the Raman scattering (3.5.16) perpendicular to the plane of polarization of the incident laser light to the polarization component of Raman scattering parallel to the polarization component"
[def08]: ## "Raman scattering (3.5.16) caused by electronic transitions of materials"
[def09]: ## "<TERS, SERS> ratio of the Raman signal intensity measured at a given wavenumber with the metallic probe or surface present to that in the absence of the probe or surface, scaled to equal numbers of scatterers"
[def10]: ## "Raman scattering caused by two-photon excitation where two photons of the source (probe laser) are practically converted to one photon of Raman scattered light and one phonon when interacting with matter"
[def11]: ## "Raman spectroscopy (3.5.17) where special attention is paid to the polarization state of the excitation and/or scattered light"
[def12]: ## "optical amplifiers based on Raman gain"
[def13]: ## "acquisition of Raman spectra as a function of depth or material removed by sputtering in order to obtain a depth representation of the spatial distribution of a particular molecular species (as indicated by the Raman effect (3.5.16))"
[def14]: ## "operation whereby the signal intensity or signal energy (band position) of a particular Raman vibrational mode or the intensity ratio of two Raman vibrational modes are mapped by undertaking Raman spectroscopy (3.5.17) while pointwise scanning the sample surface and subsequently evaluating each pixel of the image"
[def15]: ## "vibrational spectroscopic method that measures the small difference in intensity of Raman scattered right and left circularly polarized light due to molecular chirality"
[def16]: ## "inelastic scattering of electromagnetic radiation, characterized by a loss or gain in energy of the incident photons corresponding to a energy transition within the scatterer"
[def17]: ## "spectroscopy in which the radiation scattered from a sample illuminated with monochromatic radiation is characterized by an energy loss or gain arising from rotational, or vibrational mode excitations"
[def18]: ## "tensor that describes the relationship between the electric field vectors of incident and Raman scattered light for a specific Raman-active vibrational mode where the symmetry of the Raman tensor is related to the symmetry of the underlying vibrational mode"
[def19]: ## "energy difference between inelastically scattered photon and incident photon resulting via Raman effect (3.5.16) which is equal to the energy of the associated vibrational or rotational mode"
[def20]: ## "Raman spectroscopy (3.5.17) conducted under resonant conditions, where the incident or scattered light wavelength matches an optical absorption band of the sample, under these conditions the Raman scattering cross-section of particular modes are enhanced"
[def21]: ## "Raman scattering (3.5.16) caused by rotational transitions of molecules"
[def22]: ## "Raman spectroscopy that relies on random Raman scattering (3.5.16) of a proportion of incident photons"
[def23]: ## "variant of Raman spectroscopy using multiple Raman measurements from sample locations spatially offset from one another that are scaled and subtracted from one another to produce and collect/acquire Raman spectra representing subsurface and surface molecular species"
[def24]: ## "third order nonlinear phenomenon involving a second photon where the difference in the frequency of the coinciding photons matches the vibrational/rotational frequency to be excited"
[def25]: ## "Raman scattering (3.5.16) in which the scattered photon has lower energy than the incident photon"
[def26]: ## "spectroscopy using surface-enhanced Raman scattering (3.5.27)"
[def27]: ## "enhanced Raman scattering (3.5.16) observed for molecules in close proximity to a nano-scale roughened plasmonic metal surface or to metallic nanoparticles, where Raman scattering cross-sections are many orders of magnitude greater than for the same molecules in the absence of the rough plasmonic metal surface"
[def28]: ## "surface-enhanced Raman scattering in which the energy of the incident or scattered radiation is in resonance with an optical transition in the molecule"
[def29]: ## "spectroscopy using surface-enhanced resonant Raman scattering (3.5.28)"
[def30]: ## "spectroscopy using enhanced Raman scattering (3.5.16) with a plasmonically active metallic probe tip in close proximity to a sample surface illuminated with suitably polarized monochromatic light of appropriate wavelength"
[def31]: ## "spectroscopy involving a combination of tip-enhanced Raman scattering with resonance Raman spectroscopy (3.5.20)"
[def32]: ## "Raman spectroscopy whereby one side of a sample is illuminated, and the transmitted light is collected on the other side"
[def33]: ## "relationship between intensity of Raman scattering for a particular vibrational or rotational mode and the photon energy (wavelength) of the incident light"
[def34]: ## "optical configuration where the collected light propagates in the same direction as the incident light"
[def35]: ## "branch of molecular spectroscopy techniques, measuring signals caused by vibrational transitions of molecules"
[02]: ## "Note 1 to entry: This is the typical configuration of a Raman spectrometer coupled with a microscope."
[03-1]: ## "Note 1 to entry: CARS signal is generated in samples due to third-order nonlinear susceptibility and requires three laser beams. A Stokes beam with frequency ωs, a pump beam with frequency ωp and a probe beam with frequency ωpr. When the photons of the beams interact, they produce an anti-Stokes frequency (ωpr +ωp −ωs), when the term (ωp −ωs) matches the vibrational frequency of the Raman resonance then the anti-Stokes frequency is enhanced producing an orders stronger Raman signal comparing to spontaneous Raman spectroscopy."
[03-2]: ## "Note 2 to entry: See also anti-Stokes Raman scattering (3.5.1)."
[03-3]: ## "Note 3 to entry: It is a third order process as it is the cubic component of the Taylor series expansion of the nonlinear susceptibility."
[03-4]: ## "Note 4 to entry: coherent -Stokes Raman scattering (CSRS) (3.5.5) is the opposite of CARS."
[04]: ## "Note 1 to entry: Under the family of CRS are the techniques of coherent anti-Stokes Raman scattering (CARS) (3.5.3), CSRS (3.5.5) and stimulated Raman scattering (SRS) (3.5.24)."
[05-1]: ## "Note 1 to entry: CSRS requires three laser beams, the anti-Stokes beam with frequency ωAs, a pump beam with frequency ωp and a probe beam with frequency ωpr. When the photons of the beams interact they produce a Stokes frequency (ωpr +ωp −ωΑs), w hen t he t erm (ωp −ωΑs) matches the vibrational frequency of the Raman resonance then the Stokes frequency is enhanced producing an orders stronger Raman signal comparing to spontaneous Raman spectroscopy."
[05-2]: ## "Note 2 to entry: See also Stokes Raman scattering (3.5.25)."
[05-3]: ## "Note 3 to entry: It is a third order process as it is the cubic component of the Taylor series expansion of the nonlinear susceptibility."
[05-4]: ## "Note 4 to entry: CSRS is pronounced as “scissors”."
[09-1]: ## "Note 1 to entry: The enhancement factor is the product of the contrast and the ratio of the volumes analysed without the probe and with the probe."
[09-2]: ## "Note 2 to entry: In TERS the enhancement is due to the presence of the probe and in SERS it is due to the presence of a surface that is typically metallic and contains nanoscale features."
[11]: ## "Note 1 to entry: This is typically achieved using combinations of retardation plates (3.1.34), scrambler and polarizers (3.1.28) in the optical path."
[12]: ## "Note 1 to entry: See CARS (3.5.3) and SRS (3.5.24) as examples of Raman amplification."
[13]: ## "[SOURCE: adapted from ISO 14606:2015, 3.6]"
[14]: ## "Note 1 to entry: Raman spectral mapping and Raman spectral imaging are terms that are also used."
[15]: ## "Note 1 to entry: The method probes biomolecular structure and behaviour in aqueous liquids."
[16]: ## "Note 1 to entry: Typically, monochromatic optical radiation is used to probe vibrational or rotational energy levels in a molecular or crystalline sample."
[17]: ## "[SOURCE: ISO/TS 80004‑13:2017, 3.3.1.6, modified replaced emitted by scattered]"
[19]: ## "Note 1 to entry: Raman shift is typically expressed in wavenumbers."
[22]: ## "Note 1 to entry: There is no enhancement of sensitivity. This is in contrast to stimulated Raman scattering (3.5.24)."
[23]: ## "Note 1 to entry: The method is used for chemical analysis at a sub-surface level for example inside biological material or inside packaging."
[24-1]: ## "Note 1 to entry: The difference frequency sometimes called as “beat frequency”."
[24-2]: ## "Note 2 to entry: The SRS signal strength is linearly proportional to the concentration."
[25]: ## "Note 1 to entry: See anti-Stokes Raman scattering (3.5.1)."
[26]: ## "Note 1 to entry: The acronym SERS is used for both surface-enhanced Raman scattering and spectroscopy."
[27-1]: ## "Note 1 to entry: The acronym SERS is used for both surface-enhanced Raman scattering and spectroscopy."
[27-2]: ## "Note 2 to entry: The enhancement is particularly strong for gold and silver surfaces of appropriate surface roughness, when excited by a laser at the correct wavelength."
[27-3]: ## "Note 3 to entry: Surface-enhanced Raman scattering is utilized via a tip-enhancement in TERS (3.5.30)."
[28]: ## "Note 1 to entry: The acronym SERRS is used for both surface-enhanced resonant Raman scattering and spectroscopy."
[29]: ## "Note 1 to entry: The acronym SERRS is used for both surface-enhanced resonant Raman scattering and spectroscopy."
[30]: ## "Note 1 to entry: TERS provides higher scattering cross-section and higher spatial resolution than confocal Raman microscopy."
[32]: ## "Note 1 to entry: This can be used for samples up to approximately 10 mm thick with the collected Raman signal representative of the bulk material. The thickness depends on the transmittance of the sample to the excitation light."
[34]: ## "Note 1 to entry: Compare with backscattering configuration (3.5.2)."
[35]: ## "Note 1 to entry: Raman and Infrared spectroscopy are the main representative techniques of this family, each technique being sensitive to different type of vibrations and provide complementary vibrational spectra."

**Comment by StroemPhi** [comment]
Thanks @Zack-83, I put it on the agenda for tomorrow to be discussed.

**Comment by Stefano-Luin** [comment]
Please change "emitted" photon with "exiting" photon (or something similar).
Emission usually refers to a single-photon phenomenon in which an excited state decays with emission of a photon.

**Comment by Zack-83** [comment]
As discussed, we prefer to define the full hierarchy tree starting from "Raman spectroscopy" (an assay) rather than "Raman scattering" (a phenomenon)

**Comment by StroemPhi** [comment]
closed in favor of #96 which is a duplicate

**Comment by nicocopez** [comment]
Some comments after the call today. 
- TERS is related to the setup. The tip which is used as enhancer is connected to a feedback system to control is position and it is indepentent of the sample. SERS can be part of the sample (for example, the molecules to be tested can be synthesized on the surface of nanoparticle or a substrate). However, one can also just drop molecules on top of a SERS substrate (to me, this would be more in the direction of "setup" classification).
- SERS/TERS are types of enhanced spectroscopy but they can also be called near-field (enhanced) spectroscopies or antenna-enhanced spectroscopies.
- I would include wide-field microscopy in contraposition to confocal microscopy. 
Not sure if here is the right place to mention this. So please correct me if I am wrong.
